src/components/LocationCard/LocationCard.tsx

```diff
@@ -58,42 +85,15 @@ export const LocationCard = (props) => {
 
 LocationCard.displayName = 'LocationCard';
 
-LocationCard.propTypes = {
-    /** Location id */
-    locationId: PropTypes.string,
-    /** Location card title */
-    locationTitle: PropTypes.string.isRequired,
-    /** Defines if location card has a radius slider */
-    hasRadius: PropTypes.bool,
-    /** Slider indication string for displaying its formatted value */
-    sliderLabel: PropTypes.string,
-    /** The minimum value of the distance slider */
-    minRadius: PropTypes.number,
-    /** The maximum value of the distance slider */
-    maxRadius: PropTypes.number,
-    /**
-     * Value to be added or subtracted on each step the distance slider makes.
-     * Must be greater than zero, and max - min should be evenly divisible by the step value.
-     */
-    radiusStep: PropTypes.number,
-    /** The value to be displayed in the slider */
-    distanceRadius: PropTypes.number.isRequired,
-    /** Slider value change callback */
-    onRadiusChange: PropTypes.func,
-    /** Location card delete callback */
-    onDelete: PropTypes.func,
-    /** HTML tag to be used as a wrapping component */
-    As: PropTypes.string,
-};
-
 LocationCard.defaultProps = {
```

Comment:
It seems this component is using both object default values as well as defaultProps and some props (like `As` and `onRadiusChange`) are specified redundantly.

Reply:
yes, to help out TS. There is a section about it in CONTRIBUTE.md#L94